src/CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ find_program(PYINSTALLER "pyinstaller")
 
 #if pyinstaller available add compiled version of python scripts
 if (PYINSTALLER)
+    install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/python/config.yaml DESTINATION . )
 
```

Comment:
why copying this file? in `if else` condition the file is integrated in `antares-xpansion-launcher` exe

Reply:
I had an issue where I could not start antares-launcher in my install directory. It was looking for this file.